### ISSUE TITLE
Update termsOfUse in context and remove note about bug

### ISSUE THREAD

**Issue by kdenhartog** [w3c/vc-data-model#810]
This is a follow up issue to address the changes that were discussed in #776 and temporarily noted in #787, but deferred until they could be properly addressed in V2.

The following needs to be done to address this issue.
1. remove the note added in #787
2. update the V2 context to include the `termsOfUse` term

**Comment by OR13** [comment]
v2 context includes the term: 

https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L62

Term does not dereference to a human readable definition:

https://www.w3.org/2018/credentials#termsOfUse

**Comment by OR13** [comment]
Blocked by https://github.com/w3c/vc-data-model/issues/990#issuecomment-1342312804

**Comment by OR13** [comment]
Also related https://github.com/w3c/vc-data-model/issues/764

**Comment by OR13** [comment]
@iherman assigning you since I believe you will fix all these things when you implement the redirect for v2.

**Comment by OR13** [comment]
Related https://github.com/w3c/vc-data-model/issues/966

**Comment by TallTed** [comment]
I think `termsOfUse` falls into business logic, again ... and unless we're going to define (and restrict) all its potential values (which I'm pretty sure we're not), I don't think it belongs here.

**Comment by iherman** [comment]
> @iherman assigning you since I believe you will fix all these things when you implement the redirect for v2.

If and when @manu and I finalize the automatic publication procedure for the vocabulary, then the method will become simply to add or modify an item in https://github.com/w3c/vc-data-model/blob/main/vocab/credentials/v2/vocabulary.yml. Actually, this is also the preferred way today, except that I would have to run some scripts locally before merging a PR.

(We have the prototype github action set up in https://github.com/iherman/vc-data-model-experimentation, b.t.w., ie, in https://github.com/iherman/vc-data-model-experimentation/blob/main/.github/workflows/generate-vocab-files.yml. But it relies on a slight restructuring of the repository.)

**Comment by msporny** [comment]
In order to fix this bug, we have to:

* Update to the new v2 context in the spec.
* Add the termsOfUse property to v2 context.
* Remove the note added in PR #787.

**Comment by msporny** [comment]
@TallTed wrote:
> I think `termsOfUse` falls into business logic, again ... and unless we're going to define (and restrict) all its potential values (which I'm pretty sure we're not), I don't think it belongs here.

Please define "here".

My understanding is that `termsOfUse` can be used by the presenter to inform the verifier of how they expect the presentation (and VCs) to be used. The verifier may or may not abide by those terms (for a variety of reasons).

**Comment by Sakurann** [comment]
original issue was fixed in a PR. discussion continues in the issue #1010
